tests/unit_tests/test_data_multipole.py

```diff
@@ -46,3 +46,19 @@ def test_export_to_hdf5(tmpdir, u235):
     filename = str(tmpdir.join('092235.h5'))
     u235.export_to_hdf5(filename)
     assert os.path.exists(filename)
+
+
+def test_from_endf():
+    pytest.importorskip('vectfit')
+    endf_data = os.environ['OPENMC_ENDF_DATA']
+    endf_file = os.path.join(endf_data, 'neutrons', 'n-001_H_001.endf')
+    return openmc.data.WindowedMultipole.from_endf(
+            endf_file, wmp_options={"n_win": 400, "n_cf": 3})
```

Comment:
To make sure that all the lines of code for `if log` get run during tests, I'd suggest changing to
```suggestion
            endf_file, log=True, wmp_options={"n_win": 400, "n_cf": 3})
```

Reply:
ok